recipes/r-eoffice/meta.yaml

```
@@ -0,0 +1,108 @@
+{% set version = '0.2.2' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-eoffice
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/eoffice_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/eoffice/eoffice_{{ version }}.tar.gz
+  sha256: 44b31804cd6aef8ca9bec1535b9c4a9ec46d429ffb8c8e045ec5f6a732a5be5b
+
+build:
+  merge_build_host: True
+  number: 0
+  run_exports: '{{ pin_compatible("r-eoffice", max_pin="x.x") }}'
```

Conda Forge does not do this for R packages - that's more a Bioconda thing. Also, this package doesn't need compilation, so:

```suggestion
  noarch: generic
```